### PR TITLE
OTC-643 adding continual settlement and validation enum

### PR DESCRIPTION
**Pull request by claudio-viola**
- Added enum for CONTINUAL settlement delay
- Updated valid settlement models to include CONTINUAL GROSS BILATERAL SETTLEMENT

### CODE REVIEW

**Comment by claudio-viola** [comment]
Changing base branch to feature/OTC

File: src/enums/settlements.js [hunk 1]
```diff
@@ -47,7 +48,9 @@ const ValidSettlementModels = [
   { settlementDelay: 'IMMEDIATE', settlementGranularity: 'GROSS', settlementInterchange: 'BILATERAL' },
   { settlementDelay: 'DEFERRED', settlementGranularity: 'GROSS', settlementInterchange: 'BILATERAL' },
   { settlementDelay: 'DEFERRED', settlementGranularity: 'NET', settlementInterchange: 'BILATERAL' },
-  { settlementDelay: 'DEFERRED', settlementGranularity: 'NET', settlementInterchange: 'MULTILATERAL' }
+  { settlementDelay: 'DEFERRED', settlementGranularity: 'NET', settlementInterchange: 'MULTILATERAL' },
+  { settlementDelay: 'CONTINUAL', settlementGranularity: 'GROSS', settlementInterchange: 'BILATERAL' }
```
Comment (ndonnan) [inline src/enums/settlements.js:52]: It looks like the option is already present for CGS i.e. IMMEDIATE, GROSS, BILATERAL so this change probably isn't needed

Comment (claudio-viola) [reply src/enums/settlements.js:52]: if that is the case then the story is already done and functionality available in central-ledger.?

**Comment by claudio-viola** [comment]
closing as existing enum values suffice to implement.

**Pull request closed without merging**